.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -364,23 +425,37 @@ main() {
     NAME_SPACE="showcase-ci-nightly"
     NAME_SPACE_RBAC="showcase-rbac-nightly"
     NAME_SPACE_POSTGRES_DB="postgress-external-db-nightly"
+    NAME_SPACE_AKS="showcase-aks-ci-nightly"
+  fi
+  if [[ "$JOB_NAME" == *aks* ]]; then
+    az_login
+    az_aks_start "bsCluster" "bsRG"
+    az_aks_approuting_enable "bsCluster" "bsRG"
   fi
 
   install_oc
-  oc login --token="${K8S_CLUSTER_TOKEN}" --server="${K8S_CLUSTER_URL}"
+  oc login --token="${K8S_CLUSTER_TOKEN}" --server="${K8S_CLUSTER_URL}" --insecure-skip-tls-verify
   echo "OCP version: $(oc version)"
 
   API_SERVER_URL=$(oc whoami --show-server)
-  K8S_CLUSTER_ROUTER_BASE=$(oc get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')
+  if [[ "$JOB_NAME" == *aks* ]]; then
+    K8S_CLUSTER_ROUTER_BASE=$(kubectl get svc nginx --namespace app-routing-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  else
+    K8S_CLUSTER_ROUTER_BASE=$(oc get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')
+  fi
 
   echo "K8S_CLUSTER_ROUTER_BASE : $K8S_CLUSTER_ROUTER_BASE"
 
   ENCODED_API_SERVER_URL=$(echo "${API_SERVER_URL}" | base64)
   ENCODED_CLUSTER_NAME=$(echo "my-cluster" | base64)
 
   initiate_deployments
-  check_and_test "${RELEASE_NAME}" "${NAME_SPACE}"
-  check_and_test "${RELEASE_NAME_RBAC}" "${NAME_SPACE_RBAC}"
+  if [[ "$JOB_NAME" == *aks* ]]; then
+    check_and_test "${RELEASE_NAME}" "${NAME_SPACE_AKS}"
```

Comment:
I am not seeing the rbac tests here : `check_and_test "${RELEASE_NAME_RBAC}" "${NAME_SPACE_RBAC}"`

Reply:
I initially wanted to enable just non-RBAC tests (the requirement was just for installation on AKS) and skip RBAC.
But I will try to enable the RBAC ones also (if I don't encounter any issue) and let you know.